scene/resources/3d/importer_mesh.cpp

```diff
@@ -136,7 +137,7 @@ void ImporterMesh::Surface::_split_normals(Array &r_arrays, const LocalVector<in
 
 void ImporterMesh::add_blend_shape(const String &p_name) {
 	ERR_FAIL_COND(surfaces.size() > 0);
-	blend_shapes.push_back(p_name);
+	blend_shapes.push_back(AnimationLibrary::validate_library_name(p_name));
```

Comment:
This should only replace ":". The other characters in validate library do not affect blendshapes

Reply:
I think it is not safe to allow characters that are not allowed as custom properties. However, if we did that, we would have to replace other characters such as `(` and `{`. For now, I'm okay with just validate with a colon.